production/helm/loki/README.md

```diff
@@ -1,69 +1,406 @@
-# ⚠️  DEPRECATED - Loki Helm Chart
+# loki-simple-scalable
```

Comment:
```suggestion
# Loki Simple Scalable
```

Reply:
good catch, this file should be auto-generated. For the purpose of this PR I'm going to bring in the README from the simple scalable chart and we can fix it up later to include the single binary steps as well.